weblate/templates/translate.html

```diff
@@ -11,6 +11,7 @@
 {% compress js %}
 <script defer data-cfasync="false" src="{% static 'editor/base.js' %}{{ cache_param }}"></script>
 <script defer data-cfasync="false" src="{% static 'editor/full.js' %}{{ cache_param }}"></script>
+<script defer data-cfasync="false" src="{% static 'js/screenshots/clipboard-paste.js' %}{{ cache_param }}"></script>
```

Comment:
This file seems missing.

Reply:
This commit builds on the previous commit. The file was introduced there.

Reply:
Ah, sorry I didn't notice that GitHub showed me just a commit and not a full diff.